timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/model/GraphReadUtils.java

```diff
@@ -43,15 +41,10 @@
   }
 
   public static Optional<Collection> getCollectionByVreId(Element element, Vres mappings, String vreId) {
-
-    final List<Collection> filteredTypes = Arrays.asList(getEntityTypesOrDefault(element))
+    return Arrays.asList(getEntityTypesOrDefault(element))
```

Comment:
While were at it, we might also use Arrays.stream() (though this is not very valuable as entitytypes is 2 items long.